.config/eleventy.js

```diff
@@ -23,23 +25,9 @@ module.exports = function(eleventyConfig) {
 	});
 	
 	// header anchors
-	const anchors_plugin = require('@orchidjs/eleventy-plugin-ids');
-	eleventyConfig.addPlugin(anchors_plugin,{
-		prefix:'',
-		selectors:[
-			'.container h1',
-			'.container h2',
-			'.container h3',
-			'.container h4',
-			'.container h5',
-			'.container h6',
-			'.container td:first-child',
-		]
-	});
-
+	eleventyConfig.addPlugin(IdAttributePlugin);
```

Comment:
This doesn't support nested selectors like `.container td:first-child`. In my opinion that's a reasonable tradeoff to significantly reduce the number of dependencies used here (no more JSDOM!).